ntbs-service/Pages/Notifications/Edit/_ClinicalDetailsDates.cshtml

```diff
@@ -419,34 +419,34 @@
                 </div>
             </nhs-fieldset>
         </nhs-form-group>
-    
-        <h2>@Html.DisplayNameFor(m => m.Notification.ClinicalDetails.IsPostMortem)</h2>
-            <nhs-fieldset>
-                <div class="nhsuk-radios govuk-radios--conditional" data-module="govuk-radios">
-                    <div class="nhsuk-radios__item">
-                        <input asp-for="ClinicalDetails.IsPostMortem" id="postmortem-yes" class="nhsuk-radios__input" type="radio" value="true" data-aria-controls="conditional-postmortem-conditional">
-                        <label class="nhsuk-label nhsuk-radios__label" for="postmortem-yes">
-                            Yes
-                        </label>
-                    </div>
-                    <div class="govuk-radios__conditional govuk-radios__conditional--hidden" id="conditional-postmortem-conditional">
-                        <label class="nhsuk-label" for="postmortem-save-button"> Please record the death outcome on the treatment events page </label>
-                        <button
-                            nhs-button-type="Standard"
-                            classes="ntbsuk-button--primary"
-                            name="SaveAndRouteToTreatmentEvents"
-                            value="SaveAndRouteToTreatmentEvents"
-                            id="postmortem-save-button">
-                            Save and go to treatment events
-                        </button>
-                    </div>
-                    <div class="nhsuk-radios__item">
-                        <input asp-for="ClinicalDetails.IsPostMortem" id="postmortem-no" class="nhsuk-radios__input" type="radio" value="false">
-                        <label class="nhsuk-label nhsuk-radios__label" for="postmortem-no">
-                            No
-                        </label>
-                    </div>
-                </div>
-            </nhs-fieldset>
+
+    <nhs-fieldset>
+        <nhs-fieldset-legend nhs-legend-size="Standard">@Html.DisplayNameFor(m => m.ClinicalDetails.IsPostMortem)</nhs-fieldset-legend>
+        <div class="nhsuk-radios govuk-radios--conditional" data-module="govuk-radios">
+            <div class="nhsuk-radios__item">
+                <input asp-for="ClinicalDetails.IsPostMortem" id="postmortem-yes" class="nhsuk-radios__input" type="radio" value="true" data-aria-controls="conditional-postmortem-conditional">
+                <label class="nhsuk-label nhsuk-radios__label" for="postmortem-yes">
+                    Yes
+                </label>
+            </div>
+            <div class="govuk-radios__conditional govuk-radios__conditional--hidden" id="conditional-postmortem-conditional">
+                <p class="nhsuk-label"> Please record the death outcome on the treatment events page </p>
```

Comment:
I've actually fixed this separately as part of NTBS-2090 - but you've got there first!

Reply:
The only bit that I fixed was that it was invalid to give a `<button>` a label. It was causing the screen reader to read the label, but completely ignore the text on the button.